app/[locale]/profile/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import Unauthorized from '~/components/unauthorized';
+import { getServerAuthSession } from '~/server/auth';
+
+import NavTabs from './tabs';
+
+const ProfileLayout = async ({
+  children,
+  params: { locale },
+}: {
+  children: React.ReactNode;
+  params: { locale: string };
+}) => {
+  const session = await getServerAuthSession();
+
+  return session ? (
+    <section>
+      <div className="bg-gray-100 mt-11 flex h-full gap-2 p-16">
```

Comment:
Why do we have a nested section and div?

Reply:
Just that the code was refactored multiple times and moved between files, might've missed out on this. The nested div is the parent container of tabs and the displayed container. Changing section tag to angle brackets.